infrastructure/tari_script/src/script.rs

```diff
@@ -76,6 +77,12 @@ impl BorshDeserialize for TariScript {
     fn deserialize_reader<R>(reader: &mut R) -> Result<Self, io::Error>
     where R: io::Read {
         let len = reader.read_varint()?;
+        if len > MAX_SCRIPT_BYTES {
+            return Err(io::Error::new(
+                io::ErrorKind::InvalidInput,
+                "Larger than max covenant bytes".to_string(),
```

Comment:
```suggestion
                "Larger than max script bytes".to_string(),
```

Reply:
In consensus constants this is set to 2048 but it doesn't necessarily need to be that here

Reply:
I see it as still serving two purposes. 
The one is the max safety one for deserialization, the other one is the max allowed by the network for network consensus